recipe/meta.yaml

```
@@ -229,11 +229,9 @@ outputs:
   - name: adbc-driver-bigquery
     build:
       noarch: python
-      # XXX: punting on non-unix platforms due to general issues
-      skip: true  # [win]
-      skip: true  # [build_platform != target_platform]
-    script: build-python.sh  # [not win]
-    script: build-python.bat  # [win]
+      # noarch requires only a single build
+      skip: true  # [not linux]
```

```suggestion
      skip: true  # [not linux]
      skip: true  # [build_platform != target_platform]
```

(and also for all other builds)